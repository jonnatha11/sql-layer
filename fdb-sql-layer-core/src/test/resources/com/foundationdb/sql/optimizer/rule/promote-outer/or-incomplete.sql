SELECT * FROM customers LEFT JOIN orders ON customers.cid = orders.cid
 WHERE orders.order_date < '2000-01-01' OR customers.name = 'Jones'
