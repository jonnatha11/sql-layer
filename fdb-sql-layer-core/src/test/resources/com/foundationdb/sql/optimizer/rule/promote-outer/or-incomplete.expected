SelectQuery@1664023c
  ResultSet@47570945[cid, name, oid, cid, order_date]
    Project@2321b59a[customers.cid, customers.name, orders.oid, orders.cid, orders.order_date]
      Select@5dc1ac46[or(orders.order_date < 2000-01-01,customers.name == Jones)]
        JoinNode@561557c0(LEFT[customers.cid == orders.cid])
          TableSource@13342a67(customers)
          TableSource@45cbda0a(orders)
